spec/inputs/rabbitmq_spec.rb

```diff
@@ -54,15 +60,42 @@
       subject { hare_info }
 
       it "should set the queue correctly" do
+        register_instance
         expect(subject.queue).to eql(queue)
       end
 
       it "should set the prefetch value correctly" do
+        register_instance
         expect(channel).to have_received(:prefetch=).with(123)
       end
 
       context "with an exchange declared" do
-        let(:instance) { rabbitmq_settings.merge("exchange" => "myexchange", "exchange_type" => "fanout") }
+        @manual_register = true
```

Comment:
Having looked through this, it looks like these tests were just unfinished. Having finished them I think it's good for them to come along for the ride.